mkdocs-base.yml

```diff
@@ -185,31 +199,34 @@ nav:
       - management/extended_show_grants.md
       - management/utility_user.md
   - Security:
-      - security/pam_plugin.md
-      - security/ldap-authentication.md
-      - security/ldap-system-variables.md
       - security/selinux.md
       - security/apparmor.md
-      - security/data-at-rest-encryption.md
-      - security/using-keyring-plugin.md
-      - security/using-kmip.md
-      - security/encryption-functions.md
-      - security/using-amz-kms.md
-      - security/fido-authentication-plugin.md
-      - security/rotating-master-key.md
-      - security/encrypting-tables.md
-      - security/encrypting-tablespaces.md
-      - security/encrypting-system-tablespace.md
-      - security/encrypting-temporary-files.md
-      - security/encrypting-binlogs.md
-      - security/encrypting-redo-log.md
-      - security/encrypting-undo-tablespace.md
-      - security/encrypting-threads.md
-      - security/encrypting-doublewrite-buffers.md
-      - security/verifying-encryption.md
+      - security/pam_plugin.md
+      - security/system-variables.md
       - security/ssl-improvement.md
       - security/data-masking.md
-      - security/system-variables.md
+      - LDAP features:
+        - security/ldap-authentication.md
+        - security/ldap-system-variables.md
+      - Data-at-rest-encryption:
+        - security/data-at-rest-encryption.md
+        - security/using-keyring-plugin.md
+        - Authentication:
+          - security/using-kmip.md
```

Comment:
strange indentation

Reply:
This is required to have the third level list in the side table of contents. It works.